Packs/Armorblox/Integrations/Armorblox/Armorblox.yml

```diff
@@ -0,0 +1,84 @@
+category: Email Gateway
```

Comment:
please see the other comment regarding the category.

Reply:
Changed the category to `Analytics & SIEM`